packages/shared/src/components/cards/AcquisitionFormCard.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { ReactElement, useContext, useEffect, useState } from 'react';
+import { useMutation } from '@tanstack/react-query';
+import { useRouter } from 'next/router';
+import { Card } from './Card';
+import { Radio } from '../fields/Radio';
+import { Button, ButtonSize, ButtonVariant } from '../buttons/Button';
+import { AcquisitionChannel, updateUserAcquisition } from '../../graphql/users';
+import { MiniCloseIcon } from '../icons';
+import { OnboardingTitleGradient } from '../onboarding/common';
+import { removeQueryParam } from '../../lib/links';
+import AnalyticsContext from '../../contexts/AnalyticsContext';
+import { AnalyticsEvent, UserAcquisitionEvent } from '../../lib/analytics';
+
+const options = [
+  {
+    label: 'Referred by a friend or colleague',
+    value: AcquisitionChannel.Friend,
+  },
+  { label: 'Social media', value: AcquisitionChannel.SocialMedia },
+  { label: 'Search engine', value: AcquisitionChannel.SearchEngine },
+  { label: 'Blog, forum, or discussion', value: AcquisitionChannel.Blog },
+  {
+    label: 'Browser extension store',
+    value: AcquisitionChannel.ExtensionStore,
+  },
+  {
+    label: 'Advertisement or sponsorship',
+    value: AcquisitionChannel.Advertisement,
+  },
+  { label: 'Other', value: AcquisitionChannel.Other },
+];
+
+export const acquisitionKey = 'ua';
+
+export function AcquisitionFormCard(): ReactElement {
+  const { trackEvent } = useContext(AnalyticsContext);
+  const [isDismissed, setIsDismissed] = useState(false);
+  const [value, setValue] = useState<AcquisitionChannel>();
+  const router = useRouter();
+  const onRemoveQueryParams = () => {
+    const updatedUrl = removeQueryParam(window.location.href, acquisitionKey);
+    router.replace(updatedUrl);
+  };
+  const onDismiss = () => {
+    setIsDismissed(true);
+    trackEvent({ event_name: UserAcquisitionEvent.Dismiss });
+    onRemoveQueryParams();
+  };
+
+  const { mutateAsync, isLoading, isSuccess } = useMutation(
+    updateUserAcquisition,
+    {
+      onSuccess: () => {
+        trackEvent({
+          event_name: UserAcquisitionEvent.Submit,
+          target_id: value,
+        });
+      },
+    },
+  );
+
+  useEffect(() => {
+    trackEvent({
+      event_name: AnalyticsEvent.Impression,
+      target_type: acquisitionKey,
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why are we keeping eslint disable here?

Reply:
oversight counts as a reason? 😅

Reply:
Hahah yeah, let's remove 🙏

Reply:
Hmm, so it was serving a purpose after all 😢 
```
warning  React Hook useEffect has a missing dependency: 'trackEvent'. Either include it or remove the dependency array  react-hooks/exhaustive-deps
```

Returning it back...